src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.LeadingZeroCount.cs

```diff
@@ -68,10 +134,44 @@ public static Vector256<T> Invoke(Vector256<T> x)
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             public static Vector512<T> Invoke(Vector512<T> x)
             {
+                if (Avx512BW.IsSupported && Avx512Vbmi.IsSupported && sizeof(T) == 1)
+                {
+                    Vector512<byte> lookupVectorA =
+                        Vector512.Create((byte)8, 7, 6, 6, 5, 5, 5, 5,
+                                               4, 4, 4, 4, 4, 4, 4, 4,
+                                               3, 3, 3, 3, 3, 3, 3, 3,
+                                               3, 3, 3, 3, 3, 3, 3, 3,
+                                               2, 2, 2, 2, 2, 2, 2, 2,
+                                               2, 2, 2, 2, 2, 2, 2, 2,
+                                               2, 2, 2, 2, 2, 2, 2, 2,
+                                               2, 2, 2, 2, 2, 2, 2, 2);
+                    Vector512<byte> lookupVectorB = Vector512.Create((byte)1);
+                    Vector512<byte> bit7ZeroMask = Avx512BW.CompareLessThan(x.AsByte(), Vector512.Create((byte)128));
+                    return Avx512F.And(bit7ZeroMask, Avx512Vbmi.PermuteVar64x8x2(lookupVectorA, x.AsByte(), lookupVectorB)).As<byte, T>();
```

Comment:
I think this is the only one that shouldn't be simply Widen+Lzcnt. But it does warrant a comment elaborating on how the lookup works.

In particular, `PermuteVar64x8x2` isn't immediately obvious how it operates, so elaborating that `x` is being used as an index where bit 6 selects the table, bits 5:0 select an index in the table, and anything where bit 7 is set is zeroed is goodness.